cmd/utils/flags.go

```diff
@@ -929,11 +929,27 @@ func setEthash(ctx *cli.Context, cfg *eth.Config) {
 	}
 }
 
-func checkExclusive(ctx *cli.Context, flags ...cli.Flag) {
+func checkExclusive(ctx *cli.Context, args ...interface{}) {
 	set := make([]string, 0, 1)
-	for _, flag := range flags {
-		if ctx.GlobalIsSet(flag.GetName()) {
-			set = append(set, "--"+flag.GetName())
+	for i, arg := range args {
+		if flag, ok := arg.(cli.Flag); ok {
```

Comment:
Minor suggestion, instead of nesting when the condition holds, let's `continue` when it doens't hold. This reduces the nesting depth of the entire code by one level:

```go
flag, ok := arg.(cli.Flag)
if !ok {
	continue;
}
```